src/modules/account/__fixtures__/mockLegacy.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const mockLegacy = {
+  data: {
+    legacyAddress: '3057001998458191401L',
+    balance: '0',
```

Comment:
it can be work, I'm try to cover happy path, for later we can use `mswjs/data`, it's  kind a database to keep the state also we can generate dyta dynamically or conditionally